Detectors/Base/src/MaterialManager.cxx

```diff
@@ -405,14 +405,14 @@ void MaterialManager::loadCutsAndProcessesFromFile(const char* modname, const ch
     // apply cuts via material manager interface
     for (int i = 0; i < NCUTS; ++i) {
       if (cut[i] >= 0.) {
-        SpecialCut(modname, itmed, cutnames[i], cut[i]);
```

Comment:
Detector name and module name are probably the same thing. Only that one is passed to the function and the other is read from the cut file.
In your case, this now allows to specify cuts for ALPIDE from within MFT which is an improvement.

I am still wondering about the following:
The cuts you set for ALIPE ... will they have an impact on other users of ALPIDE such as ITS? Or can ITS define their own ALIPDE cuts?

If not ... probably we still need to go further and see ALPIDE as a template that can be customized in concrete instantiations..

I'll merge this for now but's let's keep an eye on it.

Reply:
That's right, if MFT creates the materials and defines the cuts for ALPIDE, ITS will retrieve the materials and overwrite the cuts and flags. In principle, MFT and ITS should agree on the best values to use.